src/pip/_internal/wheel.py

```diff
@@ -733,10 +733,12 @@ def _build_one_inside_env(self, req, output_dir, python_tag=None,
                     shutil.move(
                         os.path.join(temp_dir.path, wheel_name), wheel_path
                     )
+                except BaseException as exc:
+                    logger.critical('Failed to build: %s', exc)
+                    logger.debug('Exception information:', exc_info=True)
+                else:
```

Comment:
FWIW, this bit of the codebase is gonna change drastically over the coming weeks.

Reply:
Ok - But if all tests pass should/can we land this incase the coming changes take longer than expected. That does seem to happen with `pip` every now and then :)